.storybook/head.html

```diff
@@ -0,0 +1,14 @@
+<link href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css' rel='stylesheet' media="screen, print">
+<link href='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/css/patternfly.min.css' rel='stylesheet' media="screen, print">
+<link href='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/css/patternfly-additions.min.css' rel='stylesheet' media="screen, print">
+<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.min.js'></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js'></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/d3/4.3.0/d3.min.js'></script>
+<script src='https://cdnjs.cloudflare.com/ajax/libs/c3/0.4.11/c3.min.js'></script>
+<!-- <script src='https://cdnjs.cloudflare.com/ajax/libs/jed/1.1.1/jed.min.js'></script> -->
+<!-- <script src='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/js/patternfly.min.js'></script> -->
```

Comment:
do you have an alternative idea? (I'm already importing the asset pipieline application.js - however webpack ignores the //= require statements)